src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -311,7 +310,6 @@ export class HuiGlanceCard extends LitElement implements LovelaceCard {
                       <hui-timestamp-display
                         .hass=${this.hass}
                         .ts=${new Date(stateObj.state)}
-                        .format=${entityConf.format}
```

Comment:
This is not a property of the config.  It should default to relative display I think.

Reply:
Is it really necessary to break this? I'm pretty sure at some point I've seen someone suggest that you can change the time format on glance card in the same way you can for entities card.

![image](https://github.com/home-assistant/frontend/assets/32912880/5a11b257-5a55-4224-810c-f04a9fa6ad03)

Reply:
Hmm... looks like it's validated as part of the YAML struct, but the docs specifically say relative time is used.  Glance card also has no UI config per entity.

Can you add it to the docs along with `state_color` and I'll modify the type here?  UI config for glance should be a separate PR.

Reply:
Okay I reverted and added to the type, which required fixing another type, and now my head is spinning... 🤯